crates/primitives/starknet/src/tests/transaction.rs

```diff
@@ -373,3 +376,72 @@ fn test_event_wrapper_builder_with_event_content() {
 
     pretty_assertions::assert_eq!(event_wrapper, expected_event);
 }
+
+#[test]
+fn test_try_into_deploy_account_transaction() {
+    fn helper(
```

Comment:
define `helper` out of the test please

Reply:
and give it an explicit name

Reply:
this helper is very specific to this test and I don't think it will be helpful anywhere else. will give it more explicit name though.

Reply:
we never know, I prefer to keep it outside even in terms of readability :)